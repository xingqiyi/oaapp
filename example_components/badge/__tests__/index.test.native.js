import React from 'react';
import renderer from 'react-test-renderer';
// import { shallow } from 'enzyme';

import { View } from 'react-native';
import Badge from '../index';

describe('Badge', () => {
  it('renders correctly', () => {
    const wrapper = renderer.create(
      <Badge dot>
        <View style={{ width: 52, height: 52, backgroundColor: 'rgba(255, 140, 101, 0.15)' }} />
      </Badge>,
    );
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
