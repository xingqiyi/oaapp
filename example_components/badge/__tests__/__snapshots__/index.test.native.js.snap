// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Badge renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "flexDirection": "row",
      },
      undefined,
    ]
  }
>
  <View
    style={
      Array [
        undefined,
      ]
    }
  >
    <View
      style={
        Object {
          "backgroundColor": "rgba(255, 140, 101, 0.15)",
          "height": 52,
          "width": 52,
        }
      }
    />
    <View
      style={
        Array [
          Object {
            "backgroundColor": "#ff5b05",
            "borderRadius": 4,
            "height": 8,
            "position": "absolute",
            "right": -4,
            "top": -4,
            "width": 8,
          },
          undefined,
        ]
      }
    />
  </View>
</View>
`;
